app/views/shared/_cancel_action_modal.html.slim

```diff
@@ -22,7 +22,10 @@
         - if idv
           = button_to t('idv.buttons.cancel'), verify_session_path, method: :delete,
             class: 'btn btn-outline blue border border-blue rounded-lg col-12 input-submit'
-        - else
+        - elsif sign_up_init
+          = button_to cancel_link_text, destroy_user_session_path, method: :get,
```

Comment:
What do you think about having this link point directly to the home page instead of signing the user out, since they aren't signed in? Seeing the flash message "You are now signed out" when I wasn't even signed in is confusing. Also, the `destroy_user_session_path` goes through some expensive logic, so might as well make it more efficient since we know we can send them directly to the home page.